src/main/java/io/jenkins/plugins/coverage/targets/CoverageResult.java

```diff
@@ -374,6 +363,24 @@ public CoverageElement getChildElement() {
         return Collections.unmodifiableMap(aggregateResults);
     }
 
+    /**
+     * Getter for property 'deltaResults'.
+     *
+     * @return Value for property 'deltaResults'.
+     */
+    public Map<CoverageElement, Float> getDeltaResults() {
+        return Collections.unmodifiableMap(deltaResults);
+    }
+
+    /**
+     * Setter for property 'deltaResults'.
+     *
+     * @param deltaResults Value to set for property 'deltaResults'.
+     */
+    public void setDeltaResults(Map<CoverageElement, Float> deltaResults) {
+        this.deltaResults = deltaResults;
```

Comment:
Since the map is already there it would be saver to change the values of the map rather then changing the map.

Reply:
I'm not sure, if I get this point.. What exactly do you mean with this?

Reply:
Effective Java Item 50: Always make defensive copies, never store mutable objects in fields. 

```suggestion
        this.deltaResults.clear();
        this.deltaResults.addAll(deltaResults);
```